extensions/amp-story/1.0/amp-story.js

```diff
@@ -1185,6 +1185,8 @@ export class AmpStory extends AMP.BaseElement {
       /* payload */ undefined,
       {bubbles: true}
     );
+    this.viewerMessagingHandler_ &&
+      this.viewerMessagingHandler_.send('storyLoaded', dict({}));
```

Comment:
Two suggestions on the name:

1. We could try to make it more generic and dispatch an event like `initialContentLoaded` but I have no idea how we'd make this work for regular AMP so not sure it's a good idea.

2. If we make it Story specific let's make it very specific, it'd be hard to tell the difference between `documentLoaded` and `storyLoaded`. Wdyt of something that'd be like `storyContentLoaded` or `storyInitialContentLoaded` or anything you like?